core/scripts/testing/scripts.js

```diff
@@ -14,6 +14,26 @@
     document.head.appendChild(style);
   }
 
+  /**
+   * The term `palette` is used to as a param to match the
+   * Ionic docs, plus here is already a `ionic:theme` query being
+   * used for `md`, `ios`, and `ionic` themes.
+   */
+  const palette = window.location.search.match(/palette=([a-z]+)/);
```

Comment:
Why not match `ionic:palette`?

Reply:
I didn't use that structure because palette is not a key within the `IonicConfig` [interface](https://github.com/ionic-team/ionic-framework/blob/c4aa090135c5fd6ea8f4e06d7faa9b982ee94ee6/core/src/utils/config.ts#L7). If I use `ionic:palette` then it might lead to confusion that it's part of the global configs.